src/onlyspans.polyleads.ui/src/components/DocumentsManager/DocumentsTable/DocumentsTableVM.tsx

```diff
@@ -16,114 +16,52 @@ import {
   DropdownMenuTrigger,
 } from '@/components/ui/dropdown-menu';
 import { FileRecognitionStatus } from '@/data/enum/fileRecognitionStatus';
-
-const data: IDocument[] = [
-  {
-    name: 'AAA Electronic Concrete Cheese',
-    createdAt: '2006-10-13T14:46:05.818Z',
-    fileRecognitionStatus: 3,
-    resource: 'Сайт Политеха',
-    createdBy: 'Zoe',
-  },
-  {
-    name: 'BBB Refined Plastic Pizza',
-    createdAt: '2013-10-07T01:44:43.285Z',
-    fileRecognitionStatus: 1,
-    resource: 'Сайт Политеха',
-    createdBy: 'Ryley',
-  },
-  {
-    name: 'CCC Refined Cotton Gloves',
-    createdAt: '1987-08-31T15:49:22.344Z',
-    fileRecognitionStatus: 0,
-    resource: 'Телеграм',
-    createdBy: 'Lilian',
-  },
-  {
-    name: 'DDD Luxurious Steel Chair',
-    createdAt: '1995-08-22T22:47:55.552Z',
-    fileRecognitionStatus: 3,
-    resource: 'Сайт Политеха',
-    createdBy: 'Alysha',
-  },
-  {
-    name: 'AAA Electronic Concrete Cheese',
-    createdAt: '2006-10-13T14:46:05.818Z',
-    fileRecognitionStatus: 2,
-    resource: 'Сайт Политеха',
-    createdBy: 'Zoe',
-  },
-  {
-    name: 'BBB Refined Plastic Pizza',
-    createdAt: '2013-10-07T01:44:43.285Z',
-    fileRecognitionStatus: 1,
-    resource: 'Сайт Политеха',
-    createdBy: 'Ryley',
-  },
-  {
-    name: 'CCC Refined Cotton Gloves',
-    createdAt: '1987-08-31T15:49:22.344Z',
-    fileRecognitionStatus: 0,
-    resource: 'Телеграм',
-    createdBy: 'Lilian',
-  },
-  {
-    name: 'DDD Luxurious Steel Chair',
-    createdAt: '1995-08-22T22:47:55.552Z',
-    fileRecognitionStatus: 4,
-    resource: 'Сайт Политеха',
-    createdBy: 'Alysha',
-  },
-  {
-    name: 'AAA Electronic Concrete Cheese',
-    createdAt: '2006-10-13T14:46:05.818Z',
-    fileRecognitionStatus: 2,
-    resource: 'Сайт Политеха',
-    createdBy: 'Zoe',
-  },
-  {
-    name: 'BBB Refined Plastic Pizza',
-    createdAt: '2013-10-07T01:44:43.285Z',
-    fileRecognitionStatus: 1,
-    resource: 'Сайт Политеха',
-    createdBy: 'Ryley',
-  },
-  {
-    name: 'CCC Refined Cotton Gloves',
-    createdAt: '1987-08-31T15:49:22.344Z',
-    fileRecognitionStatus: 0,
-    resource: 'Телеграм',
-    createdBy: 'Lilian',
-  },
-  {
-    name: 'DDD Luxurious Steel Chair',
-    createdAt: '1995-08-22T22:47:55.552Z',
-    fileRecognitionStatus: 4,
-    resource: 'Сайт Политеха',
-    createdBy: 'Alysha',
-  },
-];
+import ServiceSymbols from '@/data/constant/ServiceSymbols';
+import type { IDocumentApi } from '@/services/api/document/documentApi';
 
 export interface IDocumentsTableVM {
-  loadDocuments: () => IDocument[];
+  loadDocuments: () => void;
+  getDocuments: () => IDocument[];
```

Comment:
```suggestion
  documents: IDocument[];
```
геттер убрать можно

или попробуй, но не уверен, что сработает
то есть private documents: IDocument[] = [] должно под сигнатуру подходить

```suggestion
  get documents(): IDocument[];
```